queries.sql

```diff
@@ -7,3 +7,53 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered=TRUE;
 SELECT * from animals WHERE name <> 'Gabumon';
 SELECT * from animals WHERE weight_kg >=10.4 and weight_kg <= 17.3;
+BEGIN;
+update animals set species='unspecified' where species is null;
+ROLLBACK;
```

Comment:
- [ ] Before rolling back, you need to confirm if the change was made by selecting species from the animal table. After rolling back, there should be another query that confirms if the change was reverted. Please kindly include the queries